TESTS/mbedtls/multi/main.cpp

```diff
@@ -163,5 +174,18 @@ utest::v1::status_t greentea_test_setup(const size_t number_of_cases) {
 Specification specification(greentea_test_setup, cases, greentea_test_teardown_handler);
 
 int main() {
-    Harness::run(specification);
+    int ret = 0;
+#if defined(MBEDTLS_PLATFORM_C)
+    mbedtls_platform_context platform_ctx;
```

Comment:
```mbedtls_platform_context``` can be defined platform specific. We don't know if it will fit stack of an embedded app. It may be more robust to declare it static.

Reply:
This declaration is consistent with other examples, and in my opinion should be up to the implementer to change, if the stack is small. I would say that this is a standard place to have the context.

Reply:
Are you suggesting that if an implementation introduces ```mbedtls_platform_context``` that is inappropriate for the stack then they should change all tests. If we anticipate that, we can avoid it by creating it ```static```.

Reply:
@AndrzejKurek Do you mean Mbed TLS internal example programs or other test programs in the Mbed OS repository?

Reply:
@hanno-arm I mean our examples. Mbed OS repository examples present very mixed approach to how and where are dependencies declared.

Reply:
@mazimkhan How do we know if someone doesn't prefer to have it on the stack instead? I can introduce your change, but I feel like it's a matter of preferences.

Reply:
I don't see the reason to have a platform structure in the first place, as opposed to having parameterless functions `mbedtls_platform_setup()` and `mbedtls_platform_teardown()` and leaving it to the implementation to allocate and maintain platform specific globals. Also, how would the platform context be actually used in the way it currently works? E.g., how would a custom `printf` implementation have access to it? 

This is an independent design discussion, though, that we should probably have outside of this PR.